### DISCUSSION TITLE
Extended Syntax For Markdown

### DISCUSSION THREAD

**Discussion by ng-chicago** [11ty/eleventy#3535] [Q&A] [answered]
I need to add a sup tag inside a table cell (with other items in the cell)
With the Extended Syntax, I would do this in the markdown: ^↑^
But that seems to not work in eleventy

Is there a way to make it work?
https://www.markdownguide.org/extended-syntax/

**Comment by ng-chicago** [comment, marked as answer]
Never mind user error.
sup tag in the cell works

**Comment by Ryuno-Ki** [comment]
(In case somebody else runs into this discussion in the future)

I'm using https://www.npmjs.com/package/markdown-it-sup with an [amended Markdown-it](https://www.11ty.dev/docs/languages/markdown/#add-your-own-plugins).
